bot.js

```diff
@@ -194,17 +194,19 @@ class Bot {
             .replace(/8/g, ':eight:')
             .replace(/9/g, ':nine:')
             .replace(/0/g, ':zero:');
+        if (options.includes("plaintext")) string = String(total); // undo all replacement
```

Comment:
Can probably do an early return before `string` is defined (like we do with discarded). If it's plaintext, it shouldn't had the critical success/failure emojis either IMHO.